Src/Raft/Path.ts

```diff
@@ -111,5 +112,22 @@ class Path {
         return new Path(process.cwd());
     }
 
+    /**
+     * Get the path for the user's home directory.
+     * @return {Path} A new path describing the user's home directory.
+     */
+    static home() : Path {
+        return new Path(os.homedir());
+    }
+
+    /**
+     * Gets the folder separator character for the given filesystem:
+     * / on *nix systems and \ on Windows.
+     * @return {string} File separator character.
+     */
+    static folderSeparator() : string {
```

Comment:
The index.js file for the System template uses it:

https://github.com/tlein/AnconaTemplateGame/pull/11/files#diff-80b8ff8583212475cefc1fbc7a9fad68R27